base/src/main/scala/co/uproot/abandon/Ast.scala

```diff
@@ -369,4 +371,9 @@ case class Scope(entries: Seq[ASTEntry], parentOpt: Option[Scope]) extends ASTEn
         includedScopes.foreach(_.checkDupes())
     }
   }
+
+  def checkUnusedSymbols() {
+    definitions.filterNot(_.isUsed)
+      .foreach(d => println(s"symbol `${d.name}` defined in ${d.pos} but never used"))
```

Comment:
This should perhaps be printed to `stderr`, not `stdout`. But I am not sure if the rest of the code shows warnings / errors to `stderr`. Ignore this comment if not.